modules/phase_field/tests/phase_field_kernels/tests

```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1                
```

Comment:
What exactly did you remove, the max_parallel? or the "lu" option in your input file? I wouldn't expect much to happen in serial. You'd have to run the test in parallel to see any differences. Stop by my cube if you have any questions.